src/components/errorbars/plot.js

```diff
@@ -117,6 +118,7 @@ module.exports = function plot(traces, plotinfo, transitionOpts) {
 
                 if(isNew) {
                     xerror = errorbar.append('path')
+                        .style('vector-effect', 'non-scaling-stroke')
```

Comment:
Something still looks off:

![peek 2017-07-19 11-15](https://user-images.githubusercontent.com/6675409/28374742-a1024b5c-6c73-11e7-93e6-452cc27a3837.gif)

Looks like the whiskers appear thicker during zooms.